src/main/java/de/codecrafter47/taboverlay/config/dsl/CustomPlaceholderConfiguration.java

```diff
@@ -317,7 +321,31 @@ public ColorAnimation() {
 
             OptionalInt finalDistance = distance;
             float finalSpeed = speed;
-            return builder.acquireData(() -> new CustomPlaceholderColorAnimation(textTemplate, colors, finalDistance, finalSpeed), TypeToken.STRING, textTemplate.requiresViewerContext());
+            return builder.acquireData(() -> new CustomPlaceholderColorAnimation(textTemplate, colors, sanitizeFormats(formats.getValue()), finalDistance, finalSpeed), TypeToken.STRING, textTemplate.requiresViewerContext());
```

Comment:
This throws a null pointer exception if formats is null.

Reply:
Fixed